app/controllers/application_controller.rb

```diff
@@ -104,18 +104,19 @@ def save_and_render(model)
   def error!(message, status)
     # If the message is a Hash of errors, we put it in standard ED form
     if message.is_a?(Hash)
-      render json: {errors: message}, status: status
+      render json: { errors: message }, status: status
     else
-      render json: {error: message}, status: status
+      render json: { error: message }, status: status
     end
   end
 
   # DEPRECATED
-  def hide_cover_image;end
+  def hide_cover_image; end
 
   def mixpanel
     if Rails.env.production?
-      @mixpanel ||= Mixpanel::Tracker.new '92b66301c752642b40ca39e718517d94', { :async => true, :env => request.env }
+      @mixpanel ||= Mixpanel::Tracker.new '92b66301c752642b40ca39e718517d94',
+                                          async: true, env: request.env
```

Comment:
It's ok. Since the first parameter is a string, and not a hash, ruby interpreter will split it into 2 method variables :)